spring-cloud-starter-stream-source-jms/pom.xml

```diff
@@ -18,13 +18,9 @@
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-jms</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.integration</groupId>
-			<artifactId>spring-integration-java-dsl</artifactId>
-		</dependency>
 		<dependency>
 			<groupId>javax.jms</groupId>
-			<artifactId>jms-api</artifactId>
+			<artifactId>javax.jms-api</artifactId>
```

Comment:
Not your code, but let's consider to fix it!
This must be in `provided` scope. In the end the user relies on some target JMS vendor and that is exactly those jars responsibility to bring JMS API together with the implementation. More over we may have some classes and packages clashes if we keep this jar in CP. 

Well, at least that is what we do in other projects.

Reply:
okay, added provided scope